translation-language-detection-api-playground/index.html

```diff
@@ -46,11 +34,13 @@ <h1>💬 Translator and Language Detector API Playground</h1>
         >Translate to
         <select id="translate">
           <option value="en">English</option>
+          <option value="fr">French</option>
           <option value="ja">Japanese</option>
+          <option selected value="pt">Portuguese</option>
           <option selected value="es">Spanish</option>
         </select></label
       >
-      <button hidden type="submit">Translate</button>
+      <button hidden type="submit" class="button-style">Translate</button>
```

Comment:
There is no "button-style" CSS class in this sample. Did you mean to add one?

Reply:
I sure did, and I'll open a separate PR which fixes both issues.

Reply:
New PR: https://github.com/GoogleChromeLabs/web-ai-demos/pull/173